pkg/costmodel/cluster.go

```diff
@@ -1105,7 +1107,7 @@ func pvCosts(diskMap map[string]*Disk, resolution time.Duration, resActiveMins,
 			}
 		}
 		s := time.Unix(int64(result.Values[0].Timestamp), 0)
-		e := time.Unix(int64(result.Values[len(result.Values)-1].Timestamp), 0).Add(resolution)
+		e := time.Unix(int64(result.Values[len(result.Values)-1].Timestamp), 0)
```

Comment:
For some reason (probably to do with inaccurate timing) we were adding 1m to the ends. Just shooting ourselves in the foot. Removing fixes the accuracy and the spammy logs.